examples/README.md

```diff
@@ -0,0 +1,36 @@
+# Examples
+
+The directory contains simple examples for OTA updates
+
+## beta
+
+This is the example app that will replace the running OTA app. Typically, any new image should contain OTA update logic.
+This app should be built before any of the ota apps can use it. 
+It is better to build in release mode to minify the application. 
+
+Build: 
+```
+RUSTFLAGS="-C strip=symbols -C debuginfo=0" cargo build --example beta --release
+espflash save-image --chip <mcu_target> ./target/<mcu_target>/release/examples/beta ./beta.bin
```

Comment:
This PR commits the `beta.bin` file. I'm guessing that's a mistake, since these instructions build it? I don't think we should carry around built binaries in the repository.

Reply:
Agreed, will amend

Reply:
`beta.bin` is still part of this PR